### PR TITLE
vo_tct: support non posix systems, support 256 colors output

### PR DESCRIPTION
**Pull request by avih**

### CODE REVIEW

File: video/out/vo_tct.c [hunk 1]
```diff
@@ -43,13 +43,16 @@
...
+#define ESC_COLOR256_BG "\e[48;5;%dm"
+#define ESC_COLOR256_FG "\e[38;5;%dm"
 #define DEFAULT_WIDTH 80
 #define DEFAULT_HEIGHT 25
 
 struct vo_tct_opts {
     int algo;
     int width;   // 0 -> default
     int height;  // 0 -> default
+    int term256;  // 0 -> true color
```
Comment (rr-) [inline video/out/vo_tct.c]: Later functions use `bool`, shouldn't this be `bool` also?

File: video/out/vo_tct.c [hunk 2]
```diff
@@ -43,13 +43,16 @@
 #define ESC_GOTOXY "\e[%d;%df"
 #define ESC_COLOR_BACKGROUND "\e[48;2;%d;%d;%dm"
 #define ESC_COLOR_FOREGROUND "\e[38;2;%d;%d;%dm"
+#define ESC_COLOR256_BG "\e[48;5;%dm"
+#define ESC_COLOR256_FG "\e[38;5;%dm"
```
Comment (avih) [inline video/out/vo_tct.c]: Sounds reasonable. Will do.

**Comment by rr-** [comment]
Other than the above commits LGTM; escaped ▄ works here too.

File: video/out/vo_tct.c (hunk 1 above)
Comment (avih) [inline video/out/vo_tct.c]: `OPT_FLAG` uses `int` rather than ` bool` as far as I can tell.

**Pull request merged**